src/routes/safe/store/actions/transactionsNew/loadAllTransactions.ts

```diff
@@ -0,0 +1,89 @@
+import axios, { AxiosResponse } from 'axios'
+
+import { getNewTransactionsServiceUriFrom, getTxServiceHost } from 'src/config'
+import { checksumAddress } from 'src/utils/checksumAddress'
+import { Transaction } from '../../models/types/transactions'
+
+export type ServiceUriParams = {
+  safeAddress: string
+  limit: number
+  offset: number
+  orderBy?: string // todo: maybe this should be key of MultiSigTransaction | keyof EthereumTransaction
+  queued?: boolean
+  trusted?: boolean
+}
+
+type EndpointResponse = {
+  count: number
+  next?: string
+  previous?: string
+  results: Transaction[]
+}
+
+const getAllTransactionsUrl = (safeAddress: string) => {
+  const host = getTxServiceHost()
+  const address = checksumAddress(safeAddress)
+  const base = getNewTransactionsServiceUriFrom(address)
+
+  return `${host}${base}`
```

Comment:
I think we're messing up URI and URL terms there: https://dev.to/flippedcoding/what-is-the-difference-between-a-uri-and-a-url-4455